<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zone_de_bataille</class>
 <widget class="QDialog" name="Zone_de_bataille">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1033</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1033</width>
    <height>3000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="Ensemble_bataille">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>-30</y>
     <width>1091</width>
     <height>621</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="Arene">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1111</width>
      <height>621</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/Arene.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Arriere">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>-50</y>
      <width>851</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 229, 229);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Hopital_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/Hopital.webp</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Hopital_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>460</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/Hopital.webp</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Hopital_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>460</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/Hopital.webp</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Pokemon_KO_1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Pokemon_KO_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Pokemon_KO_3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>460</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Pokemon_attaquant">
    <property name="geometry">
     <rect>
      <x>278</x>
      <y>280</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Adversaire">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>270</y>
      <width>161</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Joueur">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>220</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Echange">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;transparent&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Images/Echange.png</normaloff>../Images/Echange.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Neutre">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Images/Neutre.webp</normaloff>../Images/Neutre.webp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Fuir">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;transparent&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Images/Exit.png</normaloff>../Images/Exit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Type1">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Type2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>60</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="PV_adversaire">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>255</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PV_attaquant">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>255</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Pokemon_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;transparent&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>140</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Pokemon_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;transparent&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>140</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Pokemon_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;transparent&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>140</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Type1_adversaire">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>340</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Type2_adversaire">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>250</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Nom_attaquant">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>220</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Nom_adversaire">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>220</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <zorder>Arriere</zorder>
   <zorder>Arene</zorder>
   <zorder>Hopital_1</zorder>
   <zorder>Hopital_2</zorder>
   <zorder>Hopital_3</zorder>
   <zorder>Pokemon_KO_1</zorder>
   <zorder>Pokemon_KO_2</zorder>
   <zorder>Pokemon_KO_3</zorder>
   <zorder>Pokemon_attaquant</zorder>
   <zorder>Adversaire</zorder>
   <zorder>Joueur</zorder>
   <zorder>Echange</zorder>
   <zorder>Neutre</zorder>
   <zorder>Fuir</zorder>
   <zorder>Type1</zorder>
   <zorder>Type2</zorder>
   <zorder>PV_adversaire</zorder>
   <zorder>PV_attaquant</zorder>
   <zorder>Pokemon_1</zorder>
   <zorder>Pokemon_2</zorder>
   <zorder>Pokemon_3</zorder>
   <zorder>Type1_adversaire</zorder>
   <zorder>Type2_adversaire</zorder>
   <zorder>Nom_attaquant</zorder>
   <zorder>Nom_adversaire</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
